src/include/CmdArgsHelper.h

```diff
@@ -20,6 +20,7 @@ const char *cmdArgsStrAquarium = R"(Options and arguments:
 --enable-full-screen-mode       : Render aquarium in full screen mode instead of window mode.
 --integrated-gpu        : Choose integrated gpu to render the application. This is only supported on Dawn and D3D12 backend.
 --num-fish [count]      : specifies how many fishes will be rendered.
+--print-average-fps     : print avarage fps when exit the application. 
 --record-fps-frequency [count]  : Record fps every count frame and print fps log when exit.

```

Comment:
From a end user's perspective, `--record-fps-frequency` is too professional to understand. If it's decided that the program can calculate final performance data by itself instead of being customized by the user , this option should be got rid of as well.

Reply:
Done